salesagility/SuiteCRM
# Fixed #7160 - Can't disable notifications

## Description



Fixed an issue with use save not saving the "Notify on Assignment" value.

## Motivation and Context

Issue reference: #7160 

## How To Test This

1. User profile -> advanced.
2. Save "Notify on Assignment".
3. Check that the value saved correctly.

## Types of changes


### Final checklist
